src/index.ts

```diff
@@ -42,6 +48,23 @@ app.use("/js", express.static(path.join(process.cwd(), "js")));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
+// Session configuration
+app.use(
+  session({
+    secret: process.env.SESSION_SECRET || "your-secret-key-change-in-production",
+    resave: false,
+    saveUninitialized: false,
+    cookie: {
+      secure: false, // Set to true in production with HTTPS
```

Comment:
the only way to change it to TRUE on production is via env vars - let's set that up